altair/utils/display.py

```diff
@@ -161,10 +162,20 @@ def default_renderer_base(
     This renderer works with modern frontends (JupyterLab, nteract) that know
     how to render the custom VegaLite MIME type listed above.
     """
+    from altair.vegalite.v5.display import VEGA_MIME_TYPE, VEGALITE_MIME_TYPE
```

Comment:
See my comment on imports in `save.py`

Reply:
Done in 6e4c8c4fc3d93e8cbd9e296ad79c11b957729eeb